assets/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>My website</title>
+  </head>
+  <body>
+    <h1>Hello, world!</h1>
+    <p>
+      Eos assumenda ut qui non. Ut et et aut repellendus ut saepe maxime
```

Comment:
Let's make this a page with some more actual information. Eg. "This page is running WebCM, do X to have WebCM proxy your own site, use `webcm.X` to send events" etc.